src_c/rect_impl.h

```diff
@@ -594,6 +606,14 @@ RectExport_RectFromObject(PyObject *obj, InnerRect *temp)
     if (RectCheck(obj)) {
         return &((RectObject *)obj)->r;
     }
+    if (OtherRectCheck(obj)) {
+        OtherInnerRect rect = ((OtherRectObject *)obj)->r;
+        temp->x = (PrimitiveType)rect.x;
+        temp->y = (PrimitiveType)rect.y;
+        temp->w = (PrimitiveType)rect.w;
+        temp->h = (PrimitiveType)rect.h;
+        return temp;
+    }
```

Comment:
Maybe the thing Starbuck pointed out could be resolved by copying what we do in the rect case, so just a 
```suggestion
    if (OtherRectCheck(obj)) {
        return &((OtherRectObject *)obj)->r;
    }
```

Reply:
That would return a different type, in the case that an `FRect` is passed in, it would try to return an `SDL_FRect`, but the function is typed to return an `SDL_Rect`.